SConstruct

```diff
@@ -548,6 +548,8 @@ build_dir = 'build/%s/%s' % (
 
 env['ROC_BINDIR'] = '#bin/%s' % host
 env['ROC_VERSION'] = open(env.File('#.version').path).read().strip()
+env['ROC_SHA'] = os.popen("cat .git/$(cat .git/HEAD | awk '{print $2}') | cut -b1-10").read().strip()
+env['ROC_VERSION_STR'] = env['ROC_VERSION'] + "-" + env['ROC_SHA']
```

Comment:
@gavv what is the policy of using single quotes or doubles quotes for strings in python?

As I might see almost everywhere in scons you use double quotes for human-readable lines (such strings do not contain any formatting) and single quotes in all other places.

Reply:
No any special policy, I think. Just my personal habit.